### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by usb248** [godban/sidescroll.js#1]
Hi, 

Your plugin is just perfect ! Exactly what i search. (This similar plugin give the same functionality https://github.com/somewebmedia/hc-sticky with more features.)

In your doc, you don't write what is the compatibility with browser (IE version, mobile device etc ...?)

It will be very interesting if your add callback and few "command" like this : 

stop    : Stops the sticky while preserving its current position.
off     :Turnes off the sticky completely.
on  :Turns the sticky back on after above two commands.
reinit  :Recalculates sticky size. Useful after altering DOM elements inside sticky.
destroy     :Completely destroys sticky and reverts element to original state.

Thanks !

**Comment by gotbahn** [comment]
@usb248 thank you for feedback and proposals

> In your doc, you don't write what is the compatibility with browser (IE version, mobile device etc ...?)

My missing, updated this info in [README.md](https://github.com/godban/sidescroll.js/blob/master/README.md)

> It will be very interesting if your add callback and few "command" like this :
> 
> stop : Stops the sticky while preserving its current position.
> off :Turnes off the sticky completely.
> on :Turns the sticky back on after above two commands.
> reinit :Recalculates sticky size. Useful after altering DOM elements inside sticky.
> destroy :Completely destroys sticky and reverts element to original state.

Good point, will think about additional functionality for next release, added ticket for myself.
Btw if you want to speed up process and contribute some of this features, you are welcome.

**Comment by usb248** [comment]
Yes thanks, if i have time. Why not.

On tablet/mobile the scroll event is not fired continuously like it is on desktop. It is fired only when the scroll actually ends.
The resulting behaviour is that the element does not reposition immediately, but just at the end of the scroll.

Have you detect this little bug (I tested on my android phone (HTC One mini), firefox browser render with this no good behaviour ... ) ?

**Comment by gotbahn** [comment]
@usb248 yeah, saw this, sometimes u even need to do additional scroll to make fixes element stay in right position.

Didn't think about solution, rarely sidebar stays sidebar on mobile. Need to investigate this, thanks.

**Comment by usb248** [comment]
Yes rarely on mobile device but often on tablet.

Maybe a track [link](http://tjvantoll.com/2012/08/19/onscroll-event-issues-on-mobile-browsers/)

**Comment by gotbahn** [comment]
@usb248 added bug ticket
